errors/errors.js

```diff
@@ -0,0 +1,17 @@
+exports.handlePsqlErrors = (err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({message: "Bad request"})
+  }
+  else next(err)
+}
+
+exports.handleCustomErrors = (err, req, res, next) => {
+  if (err.message) {
+    res.status(err.status).send({ message: err.message})
+  }
+}
```

Comment:
As things stand here, your 500 error handler will never be reached. Think about what you need to do to continue the middleware chain.